src/problem/problems.jl

```diff
@@ -7,6 +7,20 @@ is the abstract type which defines a problem.
 """
 abstract type ProblemType end
 
+"""
+The structure 
+
+    EmptyProblem{S, XI, XT, XC, TC, T}
```

Comment:
Signature doesn't match

Reply:
I corrected the signature, i.e. removing  XT, XC, TC, T